tests/operations/search-operations.test.ts

```diff
@@ -0,0 +1,41 @@
+import { SearchHeadless } from '@yext/search-headless-react';
```

Comment:
do we want the file structure for the tests to mimic the `src` directory? in which case, this should go under `tests/utils/search-operations.test.ts`

Reply:
So, we already have a `__utils__` directory already under `tests`. I think having both a `utils` and `__utils__` would be confusing. That's why I chose `operations`.